packages/vue/src/components/molecules/SfBanner/SfBanner.vue

```diff
@@ -92,13 +108,7 @@ export default {
       default: () => ({}),
     },
   },
-  data() {
-    return {
-      isMobileView: false,
```

Comment:
if this is removed then this line is not needed `return !this.isMobileView ? "div" : this.link ? "SfLink" : "SfButton";` line 128

Reply:
done :)